recipes/polybeat/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "polybeat" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 17fc46568071985446294b703696697778954f2a8ee62577563e46a6dc3e9a68
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
+    - manim
```

Comment:
This is not listed upstream under `install_requires`. I can see in the readme that it is needed but it would be good to have it in the setup.py as well. Can you make a PR?